Gemfile

```diff
@@ -42,6 +42,8 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
   gem 'rspec-rails', '~>3.5'
+  gem 'rails-controller-testing'
```

Comment:
Ikenna, could you please update the commit message explaining why this gem was added?
Would be enough with the error message we get when running the tests without it. In the long run it is very helpful to understand why we have included every gem that is not a default in Rails.

```bash
  1) ContentItemsController GET #index renders the :index template
     Failure/Error: expect(subject).to render_template(:index)

     NoMethodError:
       assert_template has been extracted to a gem. To continue using it,
               add `gem 'rails-controller-testing'` to your Gemfile.
```